src/main/java/nl/tudelft/dnainator/Main.java

```diff
@@ -1,7 +1,8 @@
 package nl.tudelft.dnainator;
 
-import javafx.application.Application;
+import com.sun.javafx.application.LauncherImpl;
```

Comment:
I could not find another way to launch multiple classes. If you can, feel free to adjust it as using internal API is indeed not the way to go.